.github/workflows/lint.yml

```diff
@@ -1,27 +0,0 @@
-name: CI
```

Comment:
this was removed because `lint` is now a part of `test` is that accurate?

Reply:
Yeah, no need to have three separate work flows IMO, because most of the setup is in installing.
And: `lint` requires `build` too (because of microbundle bundling vue, react, and preact now)